Neuron/Sections/Wallet/View/GenerateMnemonicController.swift

```diff
@@ -11,10 +11,7 @@ import UIKit
 class GenerateMnemonicController: UIViewController, NoScreenshot, EnterBackOverlayPresentable, UIGestureRecognizerDelegate {
     var password = ""
     var walletModel = WalletModel()
-    var mnemonicStr = "" {
-        didSet {
-        }
-    }
+    var mnemonicStr = ""
```

Comment:
Better change `mnemonic` to `mnemonicTextView` and change `mnemonicStr` to `mnemonic`.